pkg/SyncService.go

```diff
@@ -500,20 +473,7 @@ func (impl *SyncServiceImpl) updateOCIRegistryChartVersions(client *registry.Cli
 			impl.logger.Errorw("error in marking latest", "err", err)
 			return err
 		}
-		latestCreated.Latest = true
```

Comment:
This logic can also be removed I guess. verify once
```
// Update latest version for the chart
	if chartVersionsCount > 0 {
		var latestFlagAppVersions []*sql.AppStoreApplicationVersion
		latestChartVersion := chartVersions[0]
		latestCreated, err := impl.appStoreApplicationVersionRepository.FindOneByAppStoreIdAndVersion(appId, latestChartVersion)
		if err != nil {
			impl.logger.Errorw("error in marking latest", "err", err)
			return err
		}
		latestFlagAppVersions = append(latestFlagAppVersions, latestCreated)
		err = impl.appStoreApplicationVersionRepository.Update(latestFlagAppVersions)
		if err != nil {
			impl.logger.Errorw("error in marking latest", "err", err)
			return err
		}
	}
	```

Reply:
verified and removed